### PR TITLE
bump cats version + fix warnings fixes #105

### LINKED ISSUE #105: Bump cats-effect to 1.0.0
We provide dependency versions in `project/Dependencies.scala` file.

### PR DESCRIPTION
**Pull request by tg44**

### CODE REVIEW

**Comment by tg44** [comment]
Any idea why metadata spec broken on travis? It works on local.

**Comment by snowplowcla** [comment]
Confirmed! @tg44  has signed the Individual Contributor License Agreement. Thanks so much

File: modules/metadata/src/test/scala/com/snowplowanalytics/snowplow/scalatracker/metadata/MetadataSpec.scala [hunk 1]
```diff
@@ -15,21 +15,21 @@ package com.snowplowanalytics.snowplow.scalatracker.metadata
...
 import com.snowplowanalytics.snowplow.scalatracker.{ClockProvider, Emitter, Tracker, UUIDProvider}
 import com.snowplowanalytics.snowplow.scalatracker.Emitter.EmitterPayload
-
 import org.specs2.Specification
 import org.specs2.mock.Mockito
 
 class MetadataSpec extends Specification with Mockito {
 
+  implicit def contextShift: ContextShift[IO] = IOContextShift.global
+  implicit val timer                          = cats.effect.IO.timer(scala.concurrent.ExecutionContext.Implicits.global)
```
Comment (BenFradet) [inline modules/metadata/src/test/scala/com/snowplowanalytics/snowplow/scalatracker/metadata/MetadataSpec.scala]: no need to repeat the package name, IO is already imported above

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 target/
+.idea
```
Comment (BenFradet) [inline .gitignore]: please keep editor-related stuff in your global gitignore :+1:

**Review by BenFradet — COMMENTED** [review]
Looks good overall, thanks! I'll have a look at the metadata specs later

**Comment by tg44** [comment]
Fixed! Btw the coverage decrease is funny... I removed (delegated) functionality so the covered code decreased but the uncovered is not :)

**Comment by BenFradet** [comment]
yup, codecov can be a bit finicky at times, btw I logged #110 for `MetadataSpec`

**Pull request merged**